src/meta/src/dashboard/mod.rs

```diff
@@ -351,7 +351,12 @@ pub(super) mod handlers {
 
         for worker_node in worker_nodes {
             let client = compute_clients.get(worker_node).await.map_err(err)?;
-            let result = client.stack_trace().await.map_err(err)?;
+            let result = client
+                .stack_trace(StackTraceRequest {
+                    actor_traces_format: Some("text".to_owned()),
```

Comment:
Please add a parameter to the meta dashboard `/await_tree` call instead. Can default to `text`

Reply:
Added. Example of a request:
```
curl "http://localhost:5691/api/monitor/await_tree/?format=text" > diagnose.txt
```